src/cli/task-help.ts

```diff
@@ -1,12 +1,16 @@
-import type { CompilerSystem, Logger } from '../declarations';
+import type { Config } from '../declarations';
+import { taskTelemetry } from './task-telemetry';
 
-export const taskHelp = (sys: CompilerSystem, logger: Logger) => {
-  const p = logger.dim(sys.details.platform === 'windows' ? '>' : '$');
+export const taskHelp = async (config: Config) => {
+  const logger = config.logger;
+  const sys = config.sys;
```

Comment:
...or even `export const taskHelp = async ({ logger, sys }: Config) => {`, but that's certainly a matter of preference.

Reply:
Reading Lar's feedback here, I decided to switch to the singleton reference here instead. Since I had already edited this task, I felt like it was a good idea to refactor the signature. I was just trying to avoid making that common place in the PR because I also edited the build task.